src/build.sh

```diff
@@ -47,6 +47,7 @@ function kernel_build()
   local arch
   local menu_config
   local doc_type
+  local llvm
```

Comment:
This is a very nice commit :)
Let's add these compiler options in the `kworkflow.config` file.